internal/server/recon/golden/resolve_entities/result.json

```diff
@@ -1,43 +1,43 @@
 {
   "resolvedEntities": [
     {
-      "sourceId": "newId/SunnyvaleId",
+      "sourceId": "newId/VietnamId",
```

Comment:
@Spaceenter can you check if the response is stable or if we need to do some sorting here?

Reply:
Looks it is sorted - https://github.com/datacommonsorg/mixer/blob/master/internal/server/recon/resolve.go#L209-L212 .. I think the new order is right?